infra/base-images/base-builder/Dockerfile

```diff
@@ -191,14 +134,25 @@ RUN cd $SRC && \
     tar -xzv --strip-components=1 -f $SRC/oss-fuzz.tar.gz && \
     rm -rf examples $SRC/oss-fuzz.tar.gz
 
-COPY cargo compile compile_afl compile_dataflow compile_libfuzzer compile_honggfuzz \
-    compile_go_fuzzer precompile_honggfuzz precompile_afl debug_afl srcmap \
-    write_labels.py bazel_build_fuzz_tests /usr/local/bin/
-
-COPY detect_repo.py /opt/cifuzz/
-COPY ossfuzz_coverage_runner.go $GOPATH
+# Do precompiles before copying other scripts for better cache efficiency.
```

Comment:
COPY ossfuzz_coverage_runner.go $GOPATH
is important to keep

Reply:
See https://github.com/google/oss-fuzz/pull/6335#issuecomment-909521828 for context.

Reply:
IGNORE THIS, i am fixing this properly now. i will fix it in base-builder-go